src/bindings/python/tests/test_onnx/test_backend.py

```diff
@@ -297,24 +297,8 @@ def expect_fail(test_case_path, xfail):  # type: (str) -> None
         "OnnxBackendNodeModelTest.test_reshape_allowzero_reordered_cpu",
     ),
     (
-        xfail_issue_63044,
-        "OnnxBackendNodeModelTest.test_tril_cpu",
-        "OnnxBackendNodeModelTest.test_tril_neg_cpu",
-        "OnnxBackendNodeModelTest.test_tril_one_row_neg_cpu",
-        "OnnxBackendNodeModelTest.test_tril_out_neg_cpu",
-        "OnnxBackendNodeModelTest.test_tril_out_pos_cpu",
-        "OnnxBackendNodeModelTest.test_tril_pos_cpu",
-        "OnnxBackendNodeModelTest.test_tril_square_cpu",
-        "OnnxBackendNodeModelTest.test_tril_square_neg_cpu",
+        xfail_issue_91490,
         "OnnxBackendNodeModelTest.test_tril_zero_cpu",
-        "OnnxBackendNodeModelTest.test_triu_cpu",
-        "OnnxBackendNodeModelTest.test_triu_neg_cpu",
-        "OnnxBackendNodeModelTest.test_triu_one_row_cpu",
-        "OnnxBackendNodeModelTest.test_triu_out_neg_out_cpu",
-        "OnnxBackendNodeModelTest.test_triu_out_pos_cpu",
-        "OnnxBackendNodeModelTest.test_triu_pos_cpu",
-        "OnnxBackendNodeModelTest.test_triu_square_cpu",
-        "OnnxBackendNodeModelTest.test_triu_square_neg_cpu",
         "OnnxBackendNodeModelTest.test_triu_zero_cpu",
```

Comment:
it looks duplicate of the previous line

Reply:
previous line refers to tril (case when upper == false) as opposed to current line (triu is a case when upper == true)

Reply:
OK